### DISCUSSION TITLE
2FA for passwords

### DISCUSSION THREAD

**Discussion by StaticDevMC** [dani-garcia/vaultwarden#1371] [Ideas]
Good day,
I had a good idea overnight, and that is that there are passwords in the organizations that are very sensitive. Therefore, I would like it if there was a function that for individual passwords you need your 2FA code to continue.

With kind regards
StaticDev

**Comment by BlackDex** [comment]
Sounds like a nice idea. But for this you need to be with the upstream project.
https://community.bitwarden.com/c/feature-requests/5
